### DISCUSSION TITLE
How to upload and store image using HONO framework

### DISCUSSION THREAD

**Discussion by BandiDhruv** [honojs/hono#2720] [Q&A]
I want to upload and store image in an s3 bucket using hono web framework, but I am not getting enough resources to do so .Could anybody help - > My objective is to store image of each user in an file storage and link that image to that user in the database

**Comment by 6km** [comment]
@BandiDhruv You can use @sor4chi's [hono-storage](https://github.com/sor4chi/hono-storage/tree/main/packages).
